### PR TITLE
Add support for Component Governance ADO Scope

### PR DESCRIPTION
**Pull request by dggsax**
According to [the Azure Dev Ops PAT documentation](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?toc=%2Fazure%2Fdevops%2Fmarketplace-extensibility%2Ftoc.json&view=azure-devops&tabs=Windows#create-a-pat), the `vso.governance` scope is supported to interact with Azure DevOps Component Governance API's.

> For a custom defined PAT, the required scope for accessing the Component Governance API, vso.governance, isn't selectable in the UI.

This PR adds `vso.governance` as a supported scope.

**Validation:** the executable was built locally and a PAT was generated with the `vso.governance` scope and confirmed that the PAT was able to be utilized with the `ComponentGovernanceHttpClient` library from ADO.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -176,7 +180,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Initial project release.
 
-[Unreleased]: https://github.com/AzureAD/microsoft-authentication-cli/compare/0.8.4...HEAD
+[Unreleased]: https://github.com/AzureAD/microsoft-authentication-cli/compare/0.8.5...HEAD
```
Comment (shalinikhare27) [inline CHANGELOG.md:183]: Please revert this change. Our release is manual and we update it right before we make a new release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.5] - 2024-03-04
+### Changed
```
Comment (shalinikhare27) [inline CHANGELOG.md:10]: We can say `Added` as there is no functionality change.

**Review by shalinikhare27 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dggsax) [reply CHANGELOG.md:183]: Ok!

File: CHANGELOG.md (hunk 2 above)
Comment (dggsax) [reply CHANGELOG.md]: Updated

**Comment by reillysiemens** [comment]
@dggsax, thanks for this submission, but we'll close this in favor of #369, which should give another way of accomplishing the same thing.
